nailgun/entities.py

```diff
@@ -1970,27 +1983,14 @@ def create_missing(self):
         super(LifecycleEnvironment, self).create_missing()
         if (self.name != 'Library' and  # pylint:disable=no-member
                 not hasattr(self, 'prior')):
-            response = client.get(
-                self.path('base'),
-                data={
-                    u'name': u'Library',
-                    # pylint:disable=no-member
-                    u'organization_id': self.organization.id,
-                },
-                **self._server_config.get_client_kwargs()
-            )
-            response.raise_for_status()
-            results = response.json()['results']
+            results = self.search({'organization'}, {u'name': u'Library'})
             if len(results) != 1:
                 raise APIResponseError(
-                    'Could not find the "Library" lifecycle environment for '
-                    'organization {0}. Search results: {1}'
+                    u'Could not find the "Library" lifecycle environment for '
+                    u'organization {}. Search results: {}'
```

Comment:
@JacobCallahan You're right, this will break 2.6 compatibility. I'll revert this change.